### PR TITLE
Adding kcctl candidate

### PR DESCRIPTION
**Pull request by gunnarmorling**
Hi, this adds a migration for kcctl.

### CODE REVIEW

**Comment by gunnarmorling** [comment]
Not sure what the build failure is about; it complains about an invalid URL for ActiveMQ, but the referenced file exists. Not related to this new migration in any case.

**Comment by gunnarmorling** [comment]
Rebased to latest master and force-pushed.

File: src/main/scala/io/sdkman/changelogs/KcctlMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    order = "001",
+    id = "001_add_kcctl_candidate",
+    author = "gunnarmorling"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "kcctl",
+      name = "kcctl",
+      description =
+        "kcctl - A modern and intuitive command line client for Kafka Connect.",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/KcctlMigrations.scala:19]: I know that I'm a PITA :smile: but can you make this a bit more detailed, please? The description is the blab displayed for your candidate when you type `sdk list` or when you visit https://sdkman.io/sdks.

I like what you have on your GitHub page. Could you perhaps adapt that as your description?

Comment (gunnarmorling) [reply src/main/scala/io/sdkman/changelogs/KcctlMigrations.scala:19]: Hhow about that:

> kcctl - a modern and intuitive command-line client for Kafka Connect. Relying on the idioms and semantics of kubectl, it allows you to register and examine connectors, delete them, restart them, etc.

Comment (gunnarmorling) [reply src/main/scala/io/sdkman/changelogs/KcctlMigrations.scala]: Went ahead and pushed that update. Let me know if you think it still needs further refinement.

Comment (marc0der) [reply src/main/scala/io/sdkman/changelogs/KcctlMigrations.scala]: Excellent, I made one minor tweak if you don't mind.

Comment (gunnarmorling) [reply src/main/scala/io/sdkman/changelogs/KcctlMigrations.scala]: Perfect, thx!

**Pull request merged**